README.md

```diff
@@ -17,66 +17,52 @@
+        clusterrolebindings,    []                 []                   [list,watch,get]
+        roles,                  []                 []                   [list,watch,get]
+        rolebindings,           []                 []                   [list,watch,get]
 
 COMMANDS:
-     remote   An authorization remote reader for Prometheus, with RBAC: [namespaces](list, watch, get)
-     agent    An authorization agent before Prometheus, with RBAC: [namespaces, secrets, serviceaccounts, clusterrole, clusterrolebindings, role, rolebindings](list, watch, get)
      help, h  Shows a list of commands or help for one command
 
 GLOBAL OPTIONS:
-   --log.json     [optional] Log as JSON
-   --log.debug    [optional] Log debug info
-   --help, -h     show help
-   --version, -v  print the version
+   --log.json                    [optional] Log as JSON
+   --log.debug                   [optional] Log debug info
+   --listen-address value        [optional] Address to listening (default: ":9090")
+   --proxy-url value             [optional] URL to proxy (default: "http://localhost:9999")
+   --read-timeout value          [optional] Maximum duration before timing out read of the request, and closing idle connections (default: 5m0s)
+   --max-connections value       [optional] Maximum number of simultaneous connections (default: 512)
+   --http-white-list value       [optional] White GET list for Prometheus HTTP request (default: "/alerts", "/graph", "/status", "/flags", "/config", "/rules", "/targets", "/version", "/service-discovery", "/metrics", "/consoles/*", "/static/*", "/user/*", "/-/healthy", "/-/ready", "/debug/*")
+   --filter-reader-labels value  [optional] Filter out the configured labels when calling '/api/v1/read' (default: "prometheus", "prometheus_replica")
+   --resync-period value         [optional] Resync period of Kubernetes watching (default: 15m0s)
+   --help, -h                    show help
+   --version, -v                 print the version
 
-
-```
-
-#### Start a remote reader
-
-```bash
-$ prometheus-auth remote start -h
-NAME:
-   prometheus-auth remote start - Start a Prometheus remote reader
-
-USAGE:
-   prometheus-auth remote start [command options] [arguments...]
-
-OPTIONS:
-   --listen.address value              [optional] Address to listen (default: ":9201")
-   --remote.read-url value             [optional] URL to read from remote (default: "http://prometheus-operated.cattle-prometheus:9090/api/v1/read")
-   --remote.read-timeout value         [optional] Timeout to read from remote (default: 5s)
-   --filter.external-labels-key value  [optional] Filter out the keys of configured 'externalLabels' before reading remote (default: "prometheus", "prometheus_replica")
-   --filter.only                       [optional] Only filter out the setting keys, but not authorization
-   --watch.resync-period value         [optional] Resync period of Kubernetes watching (default: 15m0s)
 ```
 
-#### Start a agent 
+### Metrics & Debug
```

Comment:
Try to give a list of metrics we provide. We can put it some where else and add the link here.